examples/ng_networking/Makefile

```diff
@@ -0,0 +1,29 @@
+# name of your application
+APPLICATION = ng_networking
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Specify the modules that are used
+USEMODULE += ng_netif
+USEMODULE += ng_sixlowpan_frag
+USEMODULE += ng_ipv6
```

Comment:
No… This example should also show-case non-IEEE802.15.4 devices. `ng_sixlowpan_default` is included automatically for IEEE802.15.4 devices since #2947, when `ng_ipv6_default` is present.